bokehjs/src/lib/models/transforms/interpolator.ts

```diff
@@ -83,8 +83,8 @@ export abstract class Interpolator extends Transform {
     // The following sorting code is referenced from:
     // http://stackoverflow.com/questions/11499268/sort-two-arrays-the-same-way
     const list: {x: number, y: number}[] = []
-    for (const j in tsx) {
-      list.push({x: tsx[j], y: tsy[j]})
+    for (let i = 0; i < tsx.length; i++) {
```

Comment:
I would definitely put a comment here explaining why a simpler `const j in tsx` cannot be used.

Reply:
I disagree with an inline comment, but I would add it to a "good practices" wiki page or dev docs. However, as I mentioned in the main comment, I will disallow "unauthorized" `for-in` loops altogether in near future (at jslint level).